cardinal/ecs/world.go

```diff
@@ -277,7 +279,10 @@ func (w *World) createEntity(archetypeID storage.ArchetypeID) (storage.EntityID,
 		return 0, err
 	}
 	archetype.PushEntity(nextEntityID)
-
+	err = w.logger.LogEntity(w, zerolog.DebugLevel, nextEntityID)
```

Comment:
I would prefer if LogEntity doesn't return an error and logs its own error instead, but cc @technicallyty for his thoughts here

Reply:
Yeah, you're right. I agree your suggestion better.

Reply:
This is done.